source/common/http/http1/codec_impl.h

```diff
@@ -96,18 +97,22 @@ class StreamEncoderImpl : public StreamEncoder,
    */
   void endEncode();
 
+void encodeFormattedHeader(absl::string_view key, absl::string_view value);
```

Comment:
Can you get rid of the clang-format-off/on above, as macro escaping is turning the formatter off here. (and we don't do that any more for the stat macros)